Hello_Console_stud/Hello_Console_stud/Program.cs

```diff
@@ -0,0 +1,229 @@
+﻿using System;
+using System.Threading;
+
+namespace Hello_Console_stud
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int a;
+            try
+            {
+                do
+                {
+                        Console.ForegroundColor = ConsoleColor.White;
+                        Console.WriteLine(@"Please,  type the number:
+                    1.  f(a,b) = |a-b| (unary)
+                    2.  f(a) = a (binary)
+                    3.  music
+                    4.  morse sos
+          
+                    ");
+                    try
+                    {
+                        a = (int)uint.Parse(Console.ReadLine());
+                        switch (a)
+                        {
+                            case 1:
+                                My_strings();
```

Comment:
Should be MyString()

Reply:
This code style for naming methods was in template

Reply:
But I fixed this in commit 592bca63f86e969b28bb074d314dd91abb86c457